src/build.ts

```diff
@@ -204,7 +222,7 @@ export async function build(
   // Anything in _observablehq also needs a content hash, but anything in _npm
   // or _node does not (because they are already necessarily immutable).
   for (const path of globalImports) {
-    if (path.endsWith(".js")) continue;
+    if (path.endsWith(".js") || path.startsWith("/_duckdb/")) continue;
```

Comment:
The comment above this line is now a bit confusing:

```js
  // Copy over global assets (e.g., minisearch.json, DuckDB’s WebAssembly).
  // Anything in _observablehq also needs a content hash, but anything in _npm
  // or _node does not (because they are already necessarily immutable).
```

Here we’re expressly skipping `/_duckdb/`, but the confusing part is that this only covers DuckDB’s extensions; DuckDB’s Wasm bundles still live in `/_npm/@duckdb/duckdb-wasm/`… Maybe it’s okay though.